### DISCUSSION TITLE
excluding agencies in queries

### DISCUSSION THREAD

**Discussion by boredland** [transitland/transitland#377] [Q&A] [answered]
Hello all! We would like to exclude routes of certain agencies from our queries. 

Would it be possible to provide the complement to 'where.agency_ids: []' in the graphql api?

**Comment by drewda** [comment, marked as answer]
Hi @boredland, we are considering this. So far we have been figuring that it is simplest for clients to have their own code for filtering out results they do not need. We'll let you know if we add this functionality to the GraphQL API.

**Reply by boredland** [reply]
Clearly that is what we'll go with for now. Great to hear it is being considered! :tada:
